inspirehep/dojson/common/base.py

```diff
@@ -329,9 +329,10 @@ def deleted_records2marc(self, key, value):
 @journals.over('urls', '^856.[10_28]')
 def urls(self, key, value):
     """URL to external resource."""
-    urls = []
+    urls = self.get('urls', [])
```

Comment:
Depends if there might be `urls` key already added in `self`. Have you seen cases like that?